docs/user-manual.md

```diff
@@ -30,6 +30,17 @@ To stop the pie, follow these steps:
    $ CRONJOBS_LIST=$(kubectl -n ${NAMESPACE} get cronjobs --no-headers=true | awk '{print $1}')
    $ [ -n "${CRONJOBS_LIST}" ] && kubectl -n ${NAMESPACE} delete cronjobs ${CRONJOBS_LIST}
    ```
+4. Remove the pie's finalizers for the Pods.
```

Comment:
Why are these finalizers not removed automatically?

IMO, the natural way of deleting resources in the Kubernetes environment is to add the deletion timesamp and then wait for the finalizers to be deleted.

Reply:
Ideally, these finalizers should be removed by the pie controller. However, at this step, they won't be removed because the pie controller is already stopped at step 1. So, we need to remove them manually.

On the one hand, we may be able to stop the pie controller while there are no probe Pods. On the other hand, we may stop it before it clears the probe Pods. This step (step 4) handles this particular situation.